codebuild/e2e_buildspec.yml

```diff
@@ -0,0 +1,18 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      nodejs: 12
+  pre_build:
+    commands:
+      - npm ci
+      - npm run build
+  build:
+    commands:
+      - npm run cypress:ci
+reports:
+  JunitReportsTest:
+    files:
+      - '**/*'
+    base-directory: 'cypress/results' # Location of the reports
```

Comment:
Is there a way to have the frequency of builds in git instead of just in the AWS console? 
I have seen these notes for creating triggers with a cron expression to determine frequency, but it'd be good if we have the values in git. https://docs.aws.amazon.com/codebuild/latest/userguide/trigger-create.html

Reply:
Not sure that these can live in the buildspec as the triggers are cloudwatch events that trigger codebuild to run

Reply:
Okay, good to know, thanks. We should add docs regarding this setup - likely `simorgh-infrastructure` so we can know where we can easily view and update the frequency if required.

Reply:
Will raise a PR in the infra repo to add docs